client/src/App.js

```diff
@@ -54,9 +56,8 @@ class App extends Component {
 
     return (
       <div className="App">
-        <Navbar updateUser={this.updateUser} loggedIn={this.state.loggedIn} />
-        <div className="container">
-          {loggedIn && <p>Logged In as user: {this.state.username}</p>}
+        <Navbar updateUser={this.updateUser} loggedIn={loggedIn} />
+        <div className="container flex-box">
```

Comment:
why do you have a class as flex-box? that's strange cause flex-box is a css display: flex. Unless this class ONLY has that then this is okay.

Reply:
it's our fix for the footer (only the footer)

Reply:
Shouldn't name the classname as flex-box if it's a fix for only the footer in my opinion because you can have hundreds of components that use flex-box and this is confusing